app/models/user.rb

```diff
@@ -101,9 +101,12 @@ def with_team_kind(kind)
     end
 
     def with_all_associations_joined
-      includes(:conferences).group("conferences.id").
-      includes(:roles).group("roles.id").
-      includes(roles: :team).group("teams.id")
+      eager_load(:conferences).
```

Comment:
I read this the other day: http://blog.arkency.com/2013/12/rails4-preloading/ if that's any help. 
I myself am not sure of anything, these days ;-) so I hope @andremedeiros will come to the rescue. (No hurry, André.)